package.json

```diff
@@ -105,10 +105,10 @@
     "react-router": "^4.0.0-beta.6",
     "react-router-dom": "^4.0.0-beta.6",
     "react-virtualized-select": "^3.1.0",
-    "styled-components": "^1.0.8"
+    "styled-components": "^4.0.3"
   },
   "peerDependencies": {
-    "react": "^0.14.0 || ^15.0.0 || ^16.0.0-alpha",
-    "react-dom": "^0.14.0 || ^15.0.0 || ^16.0.0-alpha"
+    "react": ">= 16.3.0",
+    "react-dom": ">= 16.3.0"
```

Comment:
This is kind of a bummer. If anyone was pulling in third party components that use earlier versions of React, this change would break them. Meaning we'd have to put it in a major release.

Not that I'm definitely against that, it just raises the bar.

Reply:
I was using the lib in my project with with the solution you mentioned in https://github.com/bvaughn/react-presents/pull/14. But now v4 of `styled-components` is released, i want to update to in, but i cannot use `resolutions` anymore because they changed how injecting global styles work. I was planning to update only `styled-components` but they are using new context and forwaring refs, so has to also update the version of react.

Reply:
I gotcha.

Eh, I don't mind bumping the major then. Assuming you tested and this doesn't regress anything.